src/app/MessageDef/AttributePathIB.cpp

```diff
@@ -174,7 +174,10 @@ CHIP_ERROR AttributePathIB::Parser::GetListIndex(DataModel::Nullable<ListIndex>
 CHIP_ERROR AttributePathIB::Parser::GetGroupAttributePath(ConcreteDataAttributePath & aAttributePath) const
 {
     ReturnErrorOnFailure(GetCluster(&aAttributePath.mClusterId));
```

Comment:
I think we also need to check whether endpointId is there or not in group path, if it is there, it return error.

https://github.com/project-chip/connectedhomeip/issues/29465